### PR TITLE
Replace Logos and Remove Unnecessary Files

### PR DESCRIPTION
**Pull request by phelanksd**
Replace Logos and Remove Unnecessary Files

### CODE REVIEW

**Comment by buinhansinh** [comment]
@yosiyuki  could you review? For <link rel="mask-icon" , we don't have the svg file , so just keeps the old icon there.

**Comment by yosiyuki** [comment]
@ttmthanh Please review

**Comment by yosiyuki** [comment]
@phelanksd Sorry for misunderstanding. Please update following image

Screen Shot 2022-06-20 at 15 02 13

**Comment by phelanksd** [comment]
> Hi Yosiyuki, Please checkout branch Ticket-23_Replace_logo_and_remove_unnecessary_files and check again! This logo was updated.

image

**Comment by buinhansinh** [comment]
@yosiyuki @ttmthanh  is there any problem with this pull request?

**Review by yosiyuki — CHANGES_REQUESTED** [review]
1. Please rename `src/images/logo_talent_header-asi.png` and `src/images/logo_talent-asi.png` . They looks typo in filename
2. Changing header is not in this task's request.
Screen Shot 2022-06-22 at 10 10 15

**Comment by buinhansinh** [comment]
@yosiyuki 
1. "src/images/logo_talent-asi.png" will be renamed to "src/images/logo.png" -> is this ok?
2. The changes will be reverted.

**Comment by buinhansinh** [comment]
@yosiyuki @ttmthanh  could you review again? Thanks

**Comment by ttmthanh** [comment]
I think this picture (src/images/favicon/apple-touch-icon.png) should have white background because if transparent, it look ugly
- Transparent background
image
- White background
image

File: en/index.html [hunk 1]
```diff
@@ -82,7 +87,7 @@
       <div class="mx-auto">
         <div class="flex justify-between items-center ">
           <div class="flex w-full flex-wrap md:flex-nowrap border-b border-black">
-            <a href="/en" class="mx-4 items-center basis-1/5 flex">
+            <a href="#" class="mx-4 items-center basis-1/5 flex">
               <h1 class="font-semibold text-[20px] md:text-[28px]">TalentAsiaJobs</h1>
```
Comment (ttmthanh) [inline en/index.html]: Why you change this in this file, `index.html` and `vn/index.html`? [line:90]

File: en/index.html [hunk 2]
```diff
@@ -82,7 +87,7 @@
       <div class="mx-auto">
         <div class="flex justify-between items-center ">
           <div class="flex w-full flex-wrap md:flex-nowrap border-b border-black">
-            <a href="/en" class="mx-4 items-center basis-1/5 flex">
+            <a href="#" class="mx-4 items-center basis-1/5 flex">
```
Comment (yosiyuki) [inline en/index.html]: I think better to have link to Google crawler

**Review by ttmthanh — CHANGES_REQUESTED** [review]

**Comment by buinhansinh** [comment]
@ttmthanh @yosiyuki  please check it again. Thanks

**Review by ttmthanh — APPROVED** [review]

**Review by yosiyuki — APPROVED** [review]

**Comment by yosiyuki** [comment]
@buinhansinh Please deploy or assign to deploy this update.

1. Ask Horioka-san how to ftp access to server
2. Based on difference https://github.com/KSDHRVN/gino-lp/pull/31/files update files on server
3. Merge PR

**Comment by buinhansinh** [comment]
@yosiyuki can we deploy this branch to server? Or we just deploy the changes only?

**Review by buinhansinh — CHANGES_REQUESTED** [review]
The image size is quite big compared to its size, this heavily affects the performance of webpage on mobile. Please optimize the image size.

**Review by buinhansinh — APPROVED** [review]

**Pull request merged**